app/views/admin/claims/show.html.erb

```diff
@@ -12,11 +18,22 @@
 
     <%= render partial: "answer_section", locals: {heading: "Submission details", answers: admin_submission_details(@claim)} %>
 
+    <% if @claim.payroll_gender_missing? %>
+      <div class="govuk-warning-text">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <strong class="govuk-warning-text__text">
+          <span class="govuk-warning-text__assistive">Warning</span>
+          This claim cannot be approved.
```

Comment:
I think it might be worth having the same text here as in the flash box.
This claim cannot be approved, the payroll gender is missing and the claim will need to be referred